PlaceRouteHierFlow/Makefile

```diff
@@ -11,8 +11,12 @@ BOOST_LP?=/usr/include
 GTEST_DIR?=/opt/googletest/googletest
 
 NOPYBIND11?=0
-COVERAGE?=
-OPTS?=-g -O3
+COVERAGE?=--coverage
+ifeq ($(COVERAGE),)
+  OPTS?=-O3
+else
+  OPTS?=-g -O3
+endif
```

Comment:
@Lastdayends @stevenmburns I was trying to enable coverage by default when I noticed we are actually running coverage (--coverage) & debugging(-g) with optimization (-O3). This is not very safe as some uncovered lines may get marked as covered because of inling by the compiler optimizations. 

Having said that, I tried using -O0 but doing so dramatically explodes the testing time (from 3 minutes to 30 minutes) for our switched capacitor filter at least. Any thoughts?